api/src/main/java/org/pmiops/workbench/google/CloudResourceManagerServiceImpl.java

```diff
@@ -37,7 +37,7 @@ public class CloudResourceManagerServiceImpl implements CloudResourceManagerServ
   public static final String ADMIN_SERVICE_ACCOUNT_NAME = "cloud-resource-admin";
 
   public static final List<String> SCOPES =
-      Arrays.asList(CloudResourceManagerScopes.CLOUD_PLATFORM_READ_ONLY);
+      Collections.singletonList(CloudResourceManagerScopes.CLOUD_PLATFORM);
```

Comment:
The new API requires a stronger scope.  We had to update the permissions on some cloud users accordingly: https://precisionmedicineinitiative.atlassian.net/browse/PD-7677